web/src/services/UserInfo.ts

```diff
@@ -39,8 +39,8 @@ export function toString(method: SecondFactorMethod): Method2FA {
     }
 }
 
-export async function getUserInfo(): Promise<UserInfo> {
-    const res = await Get<UserInfoPayload>(UserInfoPath);
+export async function postUserInfo(): Promise<UserInfo> {
+    const res = await Post<UserInfoPayload>(UserInfoPath);
```

Comment:
If we just use the POST endpoint, why not totally remove the GET one from the backend?